src/yunohost/app.py

```diff
@@ -362,6 +362,12 @@ def app_info(app, show_status=False, raw=False):
 
         ret['upgradable'] = upgradable
         ret['change_url'] = os.path.exists(os.path.join(app_setting_path, "scripts", "change_url"))
+        
+        with open(os.path.join(APPS_SETTING_PATH, app, 'manifest.json')) as json_manifest:
+            manifest = json.load(json_manifest)
+        
+        if 'version' in manifest:
+            ret['version'] = manifest.get('version', '-')
```

Comment:
I think you can simplify those two by simply having the second line `ret['version'] = manifest.get('version', '-')` ? (`.get` already handle the case where the key doesn't exist, c.f. the second arg)

Otherwise in some cases there won't be a `version` key in `ret` which might make https://github.com/YunoHost/yunohost-admin/pull/246 buggy in its current form ?

Reply:
You're right!